waltz-test/src/main/python/waltz_ducktape/tests/performance/benchmark_test.py

```diff
@@ -18,24 +18,25 @@ def __init__(self, test_context):
         super(BenchmarkTest, self).__init__(test_context=test_context)
 
     @cluster(cluster_spec=MIN_CLUSTER_SPEC)
-    @parametrize(txn_size=512, txn_per_thread=1000, num_thread=100, interval=10, lock_pool_size=0, num_active_partitions=1, timeout=360)
-    @parametrize(txn_size=512, txn_per_thread=1000, num_thread=100, interval=20, lock_pool_size=0, num_active_partitions=1, timeout=360)
-    @parametrize(txn_size=512, txn_per_thread=2000, num_thread=50, interval=10, lock_pool_size=0, num_active_partitions=1, timeout=360)
-    @parametrize(txn_size=1024, txn_per_thread=1000, num_thread=100, interval=10, lock_pool_size=0, num_active_partitions=1, timeout=360)
-    @parametrize(txn_size=512, txn_per_thread=100, num_thread=100, interval=10, lock_pool_size=64, num_active_partitions=1, timeout=360)
-    @parametrize(txn_size=512, txn_per_thread=100, num_thread=100, interval=10, lock_pool_size=128, num_active_partitions=1, timeout=360)
-    @parametrize(txn_size=512, txn_per_thread=100, num_thread=100, interval=10, lock_pool_size=128, num_active_partitions=2, timeout=360)
-    def test_producer_performance(self, txn_size, txn_per_thread, num_thread, interval, lock_pool_size, num_active_partitions, timeout):
-        test_cmd = self.performance_cli.producer_test_cmd(txn_size, txn_per_thread, num_thread, interval, lock_pool_size, num_active_partitions)
+    @parametrize(txn_size=512, txn_per_thread=1000, num_thread=10, interval=10, lock_pool_size=0, num_active_partitions=1, timeout=420, up_to_date_watermark=None)
+    @parametrize(txn_size=512, txn_per_thread=1000, num_thread=10, interval=10, lock_pool_size=0, num_active_partitions=1, timeout=420, up_to_date_watermark=True)
```

Comment:
I don't think setting `up_to_date_watermark` variable to `true` will be of any help here as each and every test will setup/delete the waltz cluster at the beginning/end of the test.

Let's have a separate ticket to figure out how to bundle few tests to execute as part of  same cluster.